pages/pil/dashboard/index.js

```diff
@@ -68,6 +69,22 @@ module.exports = settings => {
 
   app.post('/', updateDataFromTask(sendData));
 
+  app.use((req, res, next) => {
+    res.locals.static.profile = req.profile;
+    res.locals.static.skipExemptions = get(req.session, [req.profileId, 'skipExemptions'], null);
+    res.locals.static.skipTraining = get(req.session, [req.profileId, 'skipTraining'], null);
+    res.locals.static.isAsru = req.user.profile.asruUser;
+    res.locals.static.isLicensing = req.user.profile.asruLicensing;
+    res.locals.static.pil = req.pil;
+
+    return canTransferPil(req)
+      .then(canTransfer => {
+        res.locals.static.canTransferPil = canTransfer;
+      })
+      .then(() => next())
+      .catch(next);
+  });
```

Comment:
moved the locals out of the form setup because they were not being carried over to the `/confirm` route